app/views/movies/index.html.erb

```diff
@@ -0,0 +1,22 @@
+<%= button_to "Discover Top 40 Movies", '/movies?q=top%40rated', method: :get %>
+
+<%= form_with model: nil, url: '/movies', method: :get do %>
+  <%= label_tag(:query, "Search for:") %>
+  <%= text_field_tag(:query, nil, placeholder: 'Movie Title') %>
+  <%= submit_tag("Search") %>
+<% end %>
+
+<% if @movies %>
+  <table class='top-rated'>
+    <tr>
+      <th>Title</th>
+      <th>Vote Average</th>
+    </tr>
+    <% @movies.each do |movie| %>
+    <tr class='movie-row'>
+      <td><%= link_to movie.title, "movies/#{movie.id}" %></td>
+      <td><%= movie.vote_average %></td>
+    </tr>
+    <% end %>
+  </table>
+<% end %>
```

Comment:
We should probably add and `else` statement here to display "No matching results" before we forget.

Reply:
Yes, you will want to include some type of check for 'No matching results' to cover the sad path user experience.